samples/widgets/view-layers-toggle/README.md

```diff
@@ -0,0 +1,29 @@
+# View Layers Toggle
```

Comment:
Can we use proper case for the title? So it would be View layers toggle. This would follow our other naming patterns

Reply:
fixed, thanks.